src/zoid/checkout/component.jsx

```diff
@@ -232,9 +232,15 @@ export function getCheckoutComponent() : CheckoutComponent {
                 }
             },
         
-            dimensions: isDevice()
-                ? { width:  '100%', height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` }
-                : { width:  `${ DEFAULT_POPUP_SIZE.WIDTH }px`, height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` }
+            dimensions: ({ props }) => {
```

Comment:
I don't think this is currently working since it's not part of the props object. Can you move it up into the props? That way we can leverage the zoid prop helpers. Ex:

```js
                dimensions: {
                    type:       'object',
                    required:   false,
                    decorate:   ({ props }) => {
                        if (props.dimensions) {
                            return { width: `${ props.dimensions.width }px`, height: `${ props.dimensions.height }px` };
                        } else if (isDevice()) {
                            return { width:  '100%', height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` };
                        } else {
                            return { width:  `${ DEFAULT_POPUP_SIZE.WIDTH }px`, height: `${ DEFAULT_POPUP_SIZE.HEIGHT }px` };
                        }
                    },
                    default: () => ({})
                },
```

Reply:
Ignore the above comment. I just learned that the `dimensions` function is part of zoid: https://github.com/krakenjs/zoid/commit/4d179e123f914cfa8cdfa902977f4ebf502c196d